.travis.yml

```diff
@@ -5,7 +5,7 @@ env:
         - PYTHON_VERSION=$TRAVIS_PYTHON_VERSION
         - NUMPY_VERSION=stable
         - MAIN_CMD='python setup.py'
-        - CONDA_DEPENDENCIES='xarray!=0.13.0 dask distributed toolz Cython sphinx cartopy pillow matplotlib scipy pyyaml pyproj pyresample coveralls coverage codecov behave netcdf4 h5py h5netcdf gdal rasterio imageio pyhdf mock libtiff pycoast pydecorate geoviews zarr six'
+        - CONDA_DEPENDENCIES='xarray!=0.13.0 dask distributed toolz Cython sphinx cartopy pillow matplotlib scipy pyyaml pyproj pyresample coveralls coverage codecov behave netcdf4 h5py h5netcdf gdal rasterio imageio pyhdf mock libtiff geoviews zarr six'
```

Comment:
Are these being implicitly installed with the test commands? Shouldn't these be added to the PIP_DEPENDENCIES?

Reply:
No, they are mocked now, so they aren't needed anymore fro the tests

Reply:
Oh interesting. Do we want them to be mocked?

Reply:
I think for unit tests they should. But I also think we should have system tests (running every night) what would use actual pycoast and pydecorate to make sure compatibility isn't broken.
IMHO this has the following advantages: 
- we can continue having passing tests (and thus continue developping) even when an external lib changes its API
- unit tests install and run faster
- unit tests don't need to know the internals of the external lib, so it keeps the testing code cleaner
BUT as said above, we then need to run system tests to make sure the whole system still works.